packages/arb-token-bridge-ui/src/state/cctpState.ts

```diff
@@ -615,9 +617,11 @@ export function useClaimCctp(tx: MergedTransaction) {
         receiveReceiptTx.status === 1
           ? getStandardizedTimestamp(BigNumber.from(Date.now()).toString())
           : null
-      updateTransfer({
+      updatePendingTransaction({
         ...tx,
         resolvedAt,
+        depositStatus: DepositStatus.L2_SUCCESS,
+        status: WithdrawalStatus.EXECUTED,
```

Comment:
I think we can get rid of these

Reply:
Looked into these again and we use `depositStatus` and `status` as the source of truth to display the status labels. Cctp uses those too now, so I updated those to:
```
depositStatus: tx.isWithdrawal ? undefined : DepositStatus.L2_SUCCESS,
status: tx.isWithdrawal ? WithdrawalStatus.EXECUTED : undefined,
```

So the data makes more sense